python/packages/autogen-core/docs/src/user-guide/core-user-guide/cookbook/local-llms-ollama-litellm.ipynb

```diff
@@ -65,7 +65,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def get_model_client() -> OpenAIChatCompletionClient:\n",
+    "def get_model_client() -> OpenAIChatCompletionClient:  # type: ignore\n",
```

Comment:
Seems mypy has issues with the `autogen_ext` imports in the docs, in this case it complains about the export type as it cant follow the import. Open to suggestions if this can be fixed rather than ignored.

Reply:
Happy to ignore this for now - we can do a pass over type: ignore at some point